README.md

```diff
@@ -1,3 +1,5 @@
+...SOON TO BE UPDATED...
+
 # Just Say It App - with Rails
 Just Say It is a classic example of a blog. It shows the list of posts and empower readers to interact with them by adding comments and liking posts.
 
```

Comment:
- you did a great job by adding the readme file but kindly use the one provided by microverse and customize it with you information also add the license it will make your project looks more professional :+1: